notebooks/RuClExample.ipynb

```diff
@@ -998,7 +998,12 @@
     "for i in [13]:\n",
     "#for i in range(19):\n",
     "    #defining precision required for the trotterized circuit\n",
+    "    lattice_size = 32\n",
```

Comment:
should these var definitions be outside the loop, just want to minimize allocating more memory, unless they use same memory then should be okay

Reply:
for the sake of clarity, knowing that these variables won't be redefined, its better to define the vars outside of the loop